lib/screens/home/upcoming_page_view.dart

```diff
@@ -172,7 +172,7 @@ class _UpcommingPageViewState extends State<UpcommingPageView> {
                   maxRadius: 28.0,
                   foregroundImage:  CachedNetworkImageProvider(employee?.avatarAttachmentUrl ?? ""),
                   child: Text(
-                    getInitials(employeeName, 2),
+                    getInitials(employeeName, 2).toUpperCase(),
```

Comment:
@sriLatha2328 toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.